src/index.html

```diff
@@ -1,17 +1,159 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta
-      name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
-    >
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <title>HTML Form</title>
-    <link rel="stylesheet" href="./style.css">
+    <title>The form</title>
+    <link rel="stylesheet" href="style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
-    <script type="text/javascript" src="./main.js"></script>
+    <form
+      method="POST"
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+    >
+      <fieldset class="fs1">
+        <legend>Personal information</legend>
+        <label class="label">
+          Surname:
+          <input
+          type="text"
+          name="surname"
+          minlength="3"
+          maxlength="20"
+          autocomplete="off"
+          >
+        </label>
+        <label class="label">
+          Name:
+          <input
+          type="text"
+          name="surname"
+          minlength="3"
+          maxlength="20"
+          autocomplete="off"
+          >
+        </label>
+        <label class="label">
+          How old are you?
+          <input
+          type="number"
+          name="age"
+          min="1"
+          max="100"
+          value="12"
+          >
+        </label>
+        <label class="label">
+          Full date of birth:
+          <input
+          type="date"
+          name="date_of_birth"
+          >
+        </label>
+        <label class="label">
+          I accept the terms of the agreement
+          <input
+          type="checkbox"
+          name="agree"
+          >
+        </label>
+      </fieldset>
+
+      <fieldset class="fs1">
+        <legend>Registration</legend>
+        <label class="label">
+          E-mail:
+          <input
+          type="email"
+          name="email"
+          placeholder="email@example.com"
+          >
+        </label>
+        <label class="label">
+          Password:
+          <input
+          type="password"
+          name="password"
+          >
+        </label>
+      </fieldset>
+
+      <fieldset class="fs1">
+        <legend>An interesting fact about you!</legend>
+        <label class="label">
+          Do you love cats?
+          <input
+           type="radio"
```

Comment:
each label/select should be wrapped with a label
```suggestion
        <div class="label">
          Do you love cats?
          <label>
            <input
               type="radio"
```